.gitignore

```diff
@@ -3,3 +3,6 @@
 /docs/_data/unstable
 /docs/.jekyll-metadata
 node_modules
+
+# Ignore exports folder
+/exports
```

Comment:
We don't need to add an `exports` folder, the user can choose to export the CSV to wherever they want. Let's remove this.